src/lib/types/minions.ts

```diff
@@ -667,6 +668,15 @@ export interface MortimerOptions extends ActivityTaskOptions {
 	type: 'Mortimer';
 }
 
+export interface MemoryHarvestOptions extends ActivityTaskOptions {
```

Comment:
Should comment here what these stand for.

Reply:
They have names in the activity

```ts
	let {
			userID,
			channelID,
			duration,
			e: energyItemID,
			t: harvestMethodIndex,
			wb: hasWispBuster,
			dh: hasDivineHand,
			dp: isUsingDivinationPotion
		} = data;
		```